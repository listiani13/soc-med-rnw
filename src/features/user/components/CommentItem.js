// @flow
import React from 'react';
import {View, StyleSheet} from 'react-native';
import {Text} from '../../../core-ui';
import {DARK_GREY} from '../../../constants/color';

type Props = {
  name: string,
  body: string,
};
export default function CommentItem(props: Props) {
  let {name, body} = props;
  return (
    <View style={styles.root}>
      <Text size="small" fontWeight="500">
        {`${name} `}
        <Text size="small" style={styles.title}>
          says:
        </Text>
      </Text>
      <Text size="xxsmall">{body}</Text>
    </View>
  );
}
const styles = StyleSheet.create({
  root: {
    marginBottom: 10,
  },
  title: {
    color: DARK_GREY,
  },
});
