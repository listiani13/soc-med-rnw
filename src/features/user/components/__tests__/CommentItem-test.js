// @flow
import React from 'react';
import {render} from 'react-testing-library';
import CommentItem from '../CommentItem';

describe('Comment Item component', () => {
  it('should render Comment Item', () => {
    let {container} = render(
      <CommentItem name="Tina" body="This is content" />,
    );
    expect(container).toMatchSnapshot();
  });
});
