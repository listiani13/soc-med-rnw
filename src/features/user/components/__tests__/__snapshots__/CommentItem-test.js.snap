// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment Item component should render Comment Item 1`] = `
<div>
  <div
    class="rn-alignItems-1oszu61 rn-borderTopStyle-1efd50x rn-borderRightStyle-14skgim rn-borderBottomStyle-rull8r rn-borderLeftStyle-mm0ijv rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-display-6koalj rn-flexShrink-1qe8dj5 rn-flexBasis-1mlwlqe rn-flexDirection-eqz5dr rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginLeft-11wrixw rn-marginBottom-15d164r rn-minHeight-ifefl9 rn-minWidth-bcqeeo rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-position-bnwqim rn-zIndex-1lgpqti"
  >
    <div
      class="rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-display-1471scf rn-fontFamily-14xgk7a rn-fontStyle-o11vmf rn-fontVariant-ebii48 rn-lineHeight-t9a87b rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginBottom-p1pxzi rn-marginLeft-11wrixw rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-textDecoration-bauka4 rn-whiteSpace-q42fyq rn-wordWrap-qvutc0"
      dir="auto"
      style="font-size: 15px; font-weight: 500;"
    >
      Tina 
      <span
        class="rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-color-i59aie rn-display-1471scf rn-fontFamily-poiln3 rn-fontStyle-o11vmf rn-fontVariant-ebii48 rn-lineHeight-t9a87b rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginBottom-p1pxzi rn-marginLeft-11wrixw rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-textDecoration-bauka4 rn-whiteSpace-irrty rn-wordWrap-qvutc0"
        dir="auto"
        style="font-size: 15px;"
      >
        says:
      </span>
    </div>
    <div
      class="rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-display-1471scf rn-fontFamily-14xgk7a rn-fontStyle-o11vmf rn-fontVariant-ebii48 rn-lineHeight-t9a87b rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginBottom-p1pxzi rn-marginLeft-11wrixw rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-textDecoration-bauka4 rn-whiteSpace-q42fyq rn-wordWrap-qvutc0"
      dir="auto"
      style="font-size: 10px;"
    >
      This is content
    </div>
  </div>
</div>
`;
